app/services/partners/request_create_service.rb

```diff
@@ -44,6 +40,16 @@ def call
       self
     end
 
+    def create_only
```

Comment:
We're not actually creating the request here, are we? Maybe rename to `initialize_only`?

Reply:
Renamed.